ui/pages/Project/components/ProjectPageUI.jsx

```diff
@@ -116,7 +116,10 @@ const ProjectPageUI = React.memo((props) => {
         </Grid.Column>
         <Grid.Column width={4}>
           <ProjectSection label="Collaborators" editButton={<AddProjectCollaboratorButton />}>
-            <ProjectCollaborators />
+            <p>AnVIL Collaborators</p>
+            <ProjectCollaborators anvilCollaborator />
+            <br /><p>Local Collaborators</p>
```

Comment:
I don't think you should use the work "Local" here, I think just list all the local collaborators with no heading, and then have the AnVIL collaborators below that with "AnVIL Workspace Users" as the header. Also, you should only include the anvil section if the project has a workspace associated with it, otherwise you should leave it out entirely.

Reply:
Okay. I was also very hesitating about the subtitles.